src/services/db/GitFileSystemService.ts

```diff
@@ -1236,6 +1241,131 @@ export default class GitFileSystemService {
       })
   }
 
+  // Delete multiple files
+  deleteMultipleFiles(
+    repoName: string,
+    items: Array<{ filePath: string; sha: string }>,
+    userId: SessionDataProps["isomerUserId"],
+    branchName: string
+  ): ResultAsync<string, GitFileSystemError | NotFoundError> {
+    let oldStateSha = ""
+    const efsVolPath = this.getEfsVolPathFromBranch(branchName)
+    const isStaging = this.isStagingFromBranchName(branchName)
+
+    return this.getLatestCommitOfBranch(repoName, branchName)
+      .andThen((latestCommit) => {
+        if (!latestCommit.sha) {
+          return errAsync(
+            new GitFileSystemError(
+              `Unable to find latest commit of repo: ${repoName} on branch "${branchName}"`
+            )
+          )
+        }
+        oldStateSha = latestCommit.sha as string
```

Comment:
nit: no need casing already since we have already declared in the func args

Reply:
Updated in ad2521fd!